frontend/packages/dev-console/integration-tests/tests/git-import-flow.scenario.ts

```diff
@@ -33,9 +63,169 @@ describe('git import flow', () => {
   });
 
   it('public git normal flow', async () => {
+    await switchPerspective(Perspective.Developer);
+    expect(sideHeader.getText()).toContain('Developer');
+    await navigateImportFromGit();
+    await browser.wait(until.textToBePresentInElement(importFromGitHeader, 'Import from git'), waitForElement);
+    expect(importFromGitHeader.getText()).toContain('Import from git');
+    await enterGitRepoUrl(testData.gitRepoUrl);
+    debugger;
+    await appName.click();
```

Comment:
Not sure, why this is not working on your machine.
Because it is the old code, not modified by me.. but to make sure I updated the with reusable functions and verified the code on my local

Reply:
Hmmm.